fc/jira/task.py

```diff
@@ -137,3 +148,74 @@ def _is_triage_task(self) -> bool:
 
     def _is_backlog_task(self) -> bool:
         return not self._is_triage_task()
+
+    def _search_for_triage(self) -> dict:
+        response = requests.get(self.search_url + 'project=qppfc+and+issueType="Triage+Task"+and+status+not+in+(resolved,closed)&fields=key,description',
+                                auth=HTTPBasicAuth(self.auth.username(), self.auth.password()))
+        response.raise_for_status()
+
+        return response
+
+    def _find_triage_score_parts(self, task) -> (str, str, str):
+        m = re.search('Importance: (.*)\\r\\n\\r\\nLOE: (.*)\\r\\n\\r\\nDate [N|n]eeded: (.*)\\r\\n\\r\\n', task['fields']['description'], re.MULTILINE)
+        if m is None:
+            return None
+        else:
+            return m.groups()
+
+    def _calc_triage_score(self, imp, loe, dt) -> int:
```

Comment:
Please spell out `imp` and `dt`.  Please add a comment for what `loe` stands for.